test/integration/user_management_test.rb

```diff
@@ -0,0 +1,149 @@
+require 'test_helper'
+
+class UserManagementTest < ActionDispatch::IntegrationTest
+
+  before do
+    Capybara.current_driver = :poltergeist
+    @user = create :user,
+                    name: 'john',
+                    email: 'user@dcaf.com',
+                    role: :cm
+
+    @admin = create :user,
+                    name: 'admin',
+                    email: 'admin@dcaf.com',
+                    role: :admin
+
+    @data_volunteer = create :user,
+                            name: 'volunteer',
+                            email: 'data@gmail.com',
+                            role: :data_volunteer
+
+  end
+
+  describe 'case manager' do
+    before do
+      log_in @user
+    end
+
+    it 'should not be able to access Admin links' do
+      assert has_no_link? 'Admin'
```

Comment:
can you throw a `visit users_path` in here and the data vol test as well and confirm that they're redirected back to the main dashboard, to emulate someone trying to get at this with just the url?